sermant-plugins/sermant-mq-grayscale/mq-grayscale-rocketmq-plugin/pom.xml

```diff
@@ -20,12 +20,14 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.alibaba</groupId>
-            <artifactId>fastjson</artifactId>
+            <groupId>io.sermant</groupId>
+            <artifactId>mq-config-common</artifactId>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.yaml</groupId>
-            <artifactId>snakeyaml</artifactId>
+            <groupId>io.sermant</groupId>
+            <artifactId>mq-config-service</artifactId>
+            <version>${project.version}</version>
         </dependency>
```

Comment:
you are not allowed to import service module in plugin module

Reply:
fixed